webhooks/tasks/automaticTeams.js

```diff
@@ -35,6 +36,11 @@ module.exports = {
     const eventType = data.properties.event;
     const eventAction = data.body.action;
 
+    // Someone added a team to the repo
+    if (eventType === 'team' && (eventAction === 'add_repository' || eventAction === 'added_to_repository')) {
```

Comment:
Thanks. Fixing now. Totally forgot that in ES6 (or even ES5?) world we got `indexOf` --> `includes` even for arrays!